recipes/chef_workstation.rb

```diff
@@ -6,10 +6,15 @@
 
 # TODO: set environment variables (maybe via script?) for kitchen.vcenter etc.
 chef_workstation = node['chef_work_environment']['chef_workstation']
+source = node['chef_work_environment']['source']
```

Comment:
`source` in here is a bit confusing as it is used as a property in several resources afterwards.